prisma/migrations/20250529115348_add_provider_model/migration.sql

```diff
@@ -0,0 +1,28 @@
+-- CreateTable
```

Comment:
@Xitija 
 since this is not merged yet --- only add one final migration file for all DB changes